src/Domain/Entity/ProgrammeContainer.php

```diff
@@ -21,7 +21,7 @@
     private $availableEpisodesCount;
 
     /** @var int */
-    private $availableGalleriesCount;
+    private $aggregatedGalleriesCount;
```

Comment:
Would it not be better to put all of these changes in `Programme`?

Reply:
Looks like the aggregatedGalleriesCount code can be moved inside Programme, thanks.